doc/source/user/plot_covid_1.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from numpy import ma
+
+filename = "who_covid_19_sit_rep_time_series.csv"
+
+nbcases = np.genfromtxt(filename, dtype=np.int, delimiter=",", skip_header=7, usecols=range(3, 17))
+dates = np.genfromtxt(filename, dtype=np.unicode_, delimiter=",", max_rows=1, usecols=range(3, 17))
```

Comment:
This script is failing to run in CI with an ascii decode error. I haven't tried to debug that, however `dtype=np.unicode_` looks a little fishy. Is auto-detection of the dtype failing?

Reply:
Well, if I don't specify the `dtype` dates is an array of `nan`. Is there a more appropriate choice here to specify string data?